docs/question/html.md

```diff
@@ -1,13 +1,32 @@
-# Using Markdown in questions
+# `question.html`
 
-HTML and custom elements are great for flexibility and expressiveness. However, they're not great for working with large amounts of text, formatting text, and so on. [Markdown](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet) is a lightweight plaintext markup syntax that's ideal for authoring simple but rich text. To enable this, PrairieLearn adds a special `<markdown>` tag to questions. When a `<markdown>` block is encountered, its contents are converted to HTML. Here's an example `question.html` that utilizes this element:
+## Template file
+
+All `question.html` files are rendered with the [Mustache](https://mustache.github.io/mustache.5.html) template engine. Outside direct variable substitution, you can use Mustache to do things like conditionally render elements. For example, you can use the following syntax to conditionally render a piece of HTML:
```

Comment:
I think the only other notable/relevant thing Mustache does is rendering over arrays. Consider a small example here (won't at all block on this).